log4j-core/src/main/java/org/apache/logging/log4j/core/appender/rolling/action/IfAll.java

```diff
@@ -106,7 +107,7 @@ public static void beforeFileTreeWalk(final PathCondition[] nestedConditions) {
      */
     @PluginFactory
     public static IfAll createAndCondition(
-            @PluginElement("PathConditions") final PathCondition... components) {
+            @PluginElement("PathConditions") @Required(message = "Components for IfAll condition not provided") final PathCondition... components) {
```

Comment:
Most other messages follow the "No ... provided" or "No ... provided for ..." patterns. I would keep the messages coherent.
```suggestion
            @PluginElement("PathConditions") @Required(message = "No components provided for IfAll") final PathCondition... components) {
```

Reply:
Thanks for the review @ppkarwasz , messages corrected according to mentioned patterns.